doc/man/s3fs.1

```diff
@@ -340,6 +340,12 @@ Put the debug message from libcurl when this option is specified.
 Specify "normal" or "body" for the parameter.
 If the parameter is omitted, it is the same as "normal".
 If "body" is specified, some API communication body data will be output in addition to the debug message output as "normal".
+.TP
+\fB\-o\fR set_check_cache_sigusr1 (default is stdout)
```

Comment:
I'm sorry about what do you mean about debugging knobs.
Does that mean you want to have an option that bundles other signal handlers such as SIGSEGV?
(I think it is possible to prepare an option to specify which signals are enabled in that way.)
Please elaborate on your intended idea.

Reply:
Please point this out again in the post-rebase code review.

Reply:
Sorry I mean that we should commit this PR to enable the debugging feature.  But I think a future PR should create a debug interface to s3fs so we can do things like list curl connections, walk inode cache, force fsync, etc.  This could be implemented as an HTTP server so it is easy to interact with.